### PR TITLE
Confirming recovery seed fix

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

**Comment by KedziaPawel** [comment]
is the loader displayed on the "recover" button while converting mnemonic?

**Comment by Kamill90** [comment]
> is the loader displayed on the "recover" button while converting mnemonic?

Sure

File: src/screens/RecoverySeed/RecoverySeedScreen.tsx [hunk 1]
```diff
@@ -87,19 +87,27 @@ class RecoverySeedScreen extends Component<Props, State> {
...
-      onSubmit(keyPair, mnemonic);
-    } catch (e) {
-      this.setState({ isLoading: false });
-      Alert.alert(e.message);
-    }
+    this.setState(
+      {
+        isLoading: true,
+      },
+      async () => {
```
Comment (KedziaPawel) [inline src/screens/RecoverySeed/RecoverySeedScreen.tsx:98]: unused async

File: ios/Podfile.lock [hunk 2]
```diff
@@ -722,4 +722,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 10567d1e173aae9ab5e898db3c41267100a01a22
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.2
```
Comment (KedziaPawel) [inline ios/Podfile.lock:725]: could you upgrade pod on your computer to 1.9.3? we are all the time changing it from 1.9.2 to 1.9.3 and from 1.9.3 to 1.9.2 :D

Comment (KedziaPawel) [inline ios/Podfile.lock:725]: could you upgrade pod on your computer to 1.9.3? we are all the time changing it from 1.9.2 to 1.9.3 and from 1.9.3 to 1.9.2 :D

**Pull request merged**